### PR TITLE
Implemented Recyclerview

### PR DESCRIPTION
**Pull request by priyanka0906**
- [X] I have read [the CONTRIBUTING guide](https://github.com/LouisCAD/kotlin-libraries-playground/blob/main/CONTRIBUTING.md)
- [X] In particular, I have checked the paragraph "Avoid common mistakes"

### CODE REVIEW

**Comment by priyanka0906** [comment]
@jmfayard  I have just implemented the recyclerview. Haven't added sample for coil .

File: android/src/main/java/playground/android/model.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package playground.android
+
+data class LibraryName(var name:String)
```
Comment (jmfayard) [inline android/src/main/java/playground/android/model.kt:3]: I would prefer if there was a second parameter: 
`val createIntent: (Context) -> Intent`

This way adding a library could be done at only one place in `Suppliers.libraries`, without the need to modify the recycler adapter

**Comment by jmfayard** [comment]
I wonder if it's better to have multiple activities like currently, or to use a single activity architecture, and then use what? fragments?

What do you think @LouisCAD ?

File: android/src/main/java/playground/android/model.kt (hunk 1 above)
Comment (priyanka0906) [reply android/src/main/java/playground/android/model.kt:3]: okay, I will do that

**Comment by jmfayard** [comment]
@priyanka0906 yes, please use fragments. If later I have a better idea, I will take care of doing the refactoring.

What is important for now is that thanks to your help, people will be able to contribute an Android sample

**Comment by priyanka0906** [comment]
> @priyanka0906 yes, please use fragments. If later I have a better idea, I will take care of doing the refactoring.
> 
> What is important for now is that thanks to your help, people will be able to contribute an Android sample

@jmfayard okay.

**Comment by priyanka0906** [comment]
Screenshot (88)
@jmfayard  how to inflate .kt file using inflate for contour sample, there is no xml file.

**Comment by jmfayard** [comment]
@priyanka0906  I think you simply return MainView since it extends View

kotlin-libraries-playground_–_MainView_kt__kotlin-libraries-playground_android__and_Slack___random___Zenika

**Comment by priyanka0906** [comment]
> @priyanka0906 I think you simply return MainView since it extends View
> 
> kotlin-libraries-playground_–_MainView_kt__kotlin-libraries-playground_android__and_Slack___random___Zenika

@jmfayard yes, its working. Thankyou : )

**Pull request merged**